### PR TITLE
test(e2e): add additional test cases

### PR DESCRIPTION
**Pull request by maxym-valor**
## Related Issue(s)

closes none

## Description

Added additional test cases for textarea, value help dialog, tabs
##### PR Quality

-   [x] the commit message(s) follows the guideline:
        https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
-   [x] tests for the changes that have been done
-   [x] all items on the PR Review Checklist are addressed :
        https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
-   [n/a] Run npm run build-pack-library and test in external application
-   [n/a] update `README.md`
-   [n/a] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)

### CODE REVIEW

File: e2e/wdio/core/tests/avatar.e2e-spec.ts [hunk 1]
```diff
@@ -1,12 +1,78 @@
...
         avatarPage.open();
     }, 1);
 
+    it('should check color avatars', () => {
+        for (let i = 0; i < getElementArrayLength(colorAvatar); i++) {
+            expect(getElementClass(colorAvatar, i)).toContain('accent-color');
+        }
+    });
+
+    it('should check placeholder avatars', () => {
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/avatar.e2e-spec.ts:27]: maybe should check transparent avatars?

**Review by Seamoo13 — COMMENTED** [review]
Did you also check they work on Safari?

**Comment by maxym-valor** [comment]
> Did you also check they work on Safari?

working on it 🙃

**Review by Seamoo13 — APPROVED** [review]

**Review by dimamarksman — APPROVED** [review]
Looks good
Left some thoughts

**Pull request merged**